lib/pages/child/plan_in_progress_page.dart

```diff
@@ -87,7 +88,7 @@ class _ChildPlanInProgressPageState extends State<ChildPlanInProgressPage> {
 									if (task.timer > 0) getTimeChip(task),
 									if (task.points.quantity != 0) getCurrencyChip(task)
 								],
-								actionButton:	ItemCardActionButton(color: Colors.teal, icon: Icons.play_arrow, onTapped: () => log("tapped start task")),
+								actionButton:	ItemCardActionButton(color: Colors.teal, icon: Icons.play_arrow, onTapped: () => Navigator.of(context).pushNamed(AppPage.childTaskInProgress.name, arguments: task.id)),
```

Comment:
Ach, must have forgotten when doing review of this page but as usual you are not using AppColors values. Please check and correct all colors on this page :)